modules/extension/app-schema/app-schema/src/test/java/org/geotools/appschema/filter/ToWKTTest.java

```diff
@@ -0,0 +1,3 @@
+package org.geotools.appschema.filter;
+
+public class ToWKTTest {}
```

Comment:
**This** test seems useless

Reply:
Doh! Yes, definitely useless :-D

Reply:
Sorry, indeed it wasn't supposed to be there, I've removed it.